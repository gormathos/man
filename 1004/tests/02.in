3
3.95159 -9.38366 5.79468
4
-6.59153 3.26727 7.19651 1.19054
