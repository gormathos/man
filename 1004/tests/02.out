1.00000 -0.36261 -68.55748 214.86892
364.62230 21.88051 75.42031 134.42199
66.56763 -38.90181 81.59273 -65.16873
